src/models/notifications.js

```diff
@@ -32,7 +32,12 @@ function verifySignature (notification, config) {
 
 function * processNotification (notification, ledgers, config) {
   if (notification.event === 'transfer.update') {
-    yield payments.updateTransfer(
+    const ledger = ledgers.getLedger(notification.resource.ledger)
```

Comment:
should the `ledger` field be a property on the `notification` instead of the `resource` (to support non-5b-ledgers)?